ggml-cuda.cu

```diff
@@ -6,9 +6,66 @@
 #include <atomic>
 #include <assert.h>
 
+#if defined(GGML_USE_HIPBLAS)
+#include <hip/hip_runtime.h>
+#include <hipblas/hipblas.h>
+#include <hip/hip_fp16.h>
+#include "rocblas/rocblas.h"
```

Comment:
The rocblas include should also be behind  `__HIP_PLATFORM_AMD__`.

Reply:
Originally was gonna do this, but this pull request originally had some calls like "__HIP_PLATFORM_AMD__ in the start but the other devs didn't like it"

But I guess if people don't mind now, then I don't mind either